node/cloud_functions/basicBot/README.md

```diff
@@ -0,0 +1,15 @@
+* **Cloud Functions Chat Bot**: This bot uses
```

Comment:
This README has many formatting mistakes. i.e. "Usage" is blank, Sentences have title-casing.
- Can you please preview this README to catch these errors?
- Can you use the same structure as this [README](https://github.com/gsuitedevs/hangouts-chat-samples/blob/master/java/basic-bot/README.md)